src/util.py

```diff
@@ -35,13 +35,43 @@ def add_major_minor_ticks(ax, x_minor_per_major=3, y_minor_per_major=3, labelsiz
 
         y_minor_per_major (:class:`int`):
             Number of minor ticks between adjacent major ticks on y-axis
+
+        basex (:class:`int`):
+            If the xaxis scale is logarithmic, this is the base for the logarithm. Default is base 10.
+
+        basey (:class:`int`):
+            If the yaxis scale is logarithmic, this is the base for the logarithm. Default is base 10.
+
+        linthreshx (:class:`int`):
+            A kwarg passed to SymmetricalLogLocator if the xaxis scale is
+            `symlog`. Defines the range (-x, x), within which the plot is
+            linear. This avoids having the plot go to infinity around zero.
+            Defaults to 2.
+
+        linthreshy (:class:`int`):
+            A kwarg passed to SymmetricalLogLocator if the yaxis scale is
```

Comment:
The same as above.

Reply:
The reasoning here is similar to above. `linthreshy` is passed into `tic.SymmetricalLogLocator` in line 72 as a `kwarg`.